package.json

```diff
@@ -14,7 +14,7 @@
     "local-storage-fallback": "~4.0.0",
     "lodash": "~4.17.4",
     "loglevel": "~1.5.1",
-    "p-queue": "^2.4.2",
+    "promise-queue": "github:thomasconner/promise-queue",
```

Comment:
Why is this version forked?

Reply:
`p-queue` allowed us to pause and start the queue. Since it is not able to be used due to issues on PhoneGap we need to replace it with a similar package called `promise-queue`. `promise-queue` does not support pausing and starting of the queue so a fork was made to add the functionality